Source/WebCore/platform/audio/cocoa/MediaSessionManagerCocoa.mm

```diff
@@ -607,10 +607,14 @@
 
 void MediaSessionManagerCocoa::updateNowPlayingSuppression(const NowPlayingInfo* nowPlayingInfo)
 {
-    if (!nowPlayingInfo || !nowPlayingInfo->isVideo)
+    if (!nowPlayingInfo || !nowPlayingInfo->isVideo) {
+        WTFLogAlways("MediaSessionManagerCocoa::updateNowPlayingSuppression: clearing suppressPresentationOverBundleIdentifiers (hasNowPlayingInfo=%d, isVideo=%d)", !!nowPlayingInfo, nowPlayingInfo && nowPlayingInfo->isVideo);
```

Comment:
Why use `WTFLogAlways` rather than `ALWAYS_LOG(LOGIDENTIFIER, ...)`?

Reply:
`updateNowPlayingSuppression` is a static function so `ALWAYS_LOG` didn't work, but I will use `RELEASE_LOG` so I can put this in the `Media` category.